File: Studio2015/Win32Lib/Employee.cpp
#include "stdafx.h"
#include "Employee.h"


Employee::Employee(std::string name, int age, int yearsEmployed, double baseSalary, double bonus) : Person(name, age)
{
	_yearsEmployed = yearsEmployed;
	_baseSalary = baseSalary;
	_bonus = bonus;
}

Employee::Employee() : Person()
{
}


Employee::~Employee()
{
}

double Employee::Earnings() const
{
	return (_baseSalary * (1 + 0.01 * _yearsEmployed) + _bonus);
}

void Employee::DoSomething()
{
}

File: Studio2015/CSharpLib/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSharpLib
{
    public class Employee : Person
    {
        public Employee(string name, int age) : base(name, age)
        {
        }

        public bool IsManager { get; set; }
    }
}
